.circleci/config.yml

```diff
@@ -11,10 +11,11 @@ jobs:
       - python/load-cache
       - python/install-deps
       - python/save-cache
-      - run: 
+      - run:
           command: |
-            export FLASK_APP=flaskr/app 
-            export FLASK_ENV=development 
+            export FLASK_APP=flaskr/app
+            export FLASK_ENV=development
+            export DATABASE_URL="postgresql://postgres:postgres@localhost:5432/rosetta_dev" 
```

Comment:
as discussed we can add a variable to the circle ci env vars if this turns out to be non-optimal